Assignment/solution_student/css/style.css

```diff
@@ -31,15 +31,231 @@ html {
 /* BASE STYLES
 /*******************************************/
 body {
-  color: black;
-  font-size: 16px;
-  background: white;
+  	color: #121212;
+  	font-size: 16px;
+  	font-weight: 300;
+  	font-family: 'Open Sans', sans-serif;
+  	background-color: #f0efef;
+}
+
+h1{
+	font-size: 24px;
+	font-weight: 800;
+	color: #FFF;
+	padding: 25px 10px 25px 10%;
+	margin: 0px;
+	display: inline-block;
+	float: left;
+}
+
+nav ul{
+	padding-right: 10%;
+	margin: 0px;
+	padding-top: 1em;
+	float: right;
+}
+
+nav li{
+	font-weight: 500;
+	color: #f9e42e;
+	list-style-type: none;
+	display: inline-block;
+	padding: 1em;
+}
+
+h2{
+	font-size: 24px;
+	font-weight: 500;
+	color: #FFF;
+}
+
+h3{
+	font-size: 20px;
+	font-weight: 500;
+	color: #033048;
+}
+
+h4{
+	font-size: 24px;
+	font-weight: 800;
+	text-align: center;
+	color: #f9e42e;
+}
+
+h5{
+	font-size: 18px;
+	font-weight: 500;
+	text-align: center;
+	color: #fff;
+}
+
+a{
+	font-size: 14px;
+	font-weight: 400;
+	border-radius: 5px; 
+	text-decoration: none;
+	color: #f9e42e;
+}
+
+img {
+	max-width: 100%;
+}
+
+article{
+	width: 60%;
+	padding: 25px 10px 10px 10%;
+	float: left;
+}
+
+article h1{
+	font-size: 40px;
+	font-weight: 500;
+	color: #121212;
+	padding-left: 0px;
+}
+
+article footer{
+	text-align: right;
+	padding-bottom: 15px;
+	font-size: 24px;
+	font-weight: 500;
+}
+
+article p{
+	padding: 25px 0px 25px 0px;
+}
+
+#sidebar h1{
+	font-size: 32px;
+	font-weight: 800;
+	color: #033048;
+	padding-top: 50px;
+	padding-left: 10%;
+	width: 100%;
+	padding-right: 20%;
+
+}
+#sidebar li, p{
```

Comment:
I could be mistaken, but I get the feeling that what you were trying to do here is target both `li` and `p` elements that are children of the `#sidebar` container, but what you actually did is target all `li` elements that are children of the `#sidebar` container **and** _all_ `p` elements. When you chain selectors together with a comma a completely new selector starts after the comma, so you would need to have written `#sidebar li, #sidebar p` if you wanted to in fact target only those elements inside of the `#sidebar` container. Does that make sense?